MIGRATION.md

```diff
@@ -537,3 +556,652 @@ import { Grid, GridItem } from '@contentful/f36-components';
+_Note: This example would render only the input and the `labelText` would be set as the `aria-label`._
+
+You will do:
+
+```jsx static=true
+import { Checkbox } from '@contentful/f36-components';
+
+// This would render the exact same as on version 3
+<Checkbox id="checkbox" name="checkbox" aria-label="some label" />
+
+// You can also pass the label and help text to the checkbox
+<Checkbox id="checkbox" name="checkbox" helpText="Some help text">
+  This label will be rendered at the side of the checkbox
+</Checkbox>
+```
+
+We also introduced API changes for the checkbox. This is an overview of the renamed properties:
+
+- `required` was renamed to `isRequired`;
+- `labelText` should be replaced by `aria-label` when not passing a label to the checkbox;
+- `checked` was renamed to `isChecked`;
+- `disabled` was renamed to `isDisabled`;
+- `indeterminate` was renamed to `isIndeterminate`.
+
+#### How to migrate your checkbox components
+
+The migration for the version 3 `Checkbox` component needs to be manual as it depends on the context that the component is being used, for example:
+
+```jsx static=true
+import {
+  Checkbox,
+  Box,
+  FormLabel,
+} from '@contentful/forma-36-react-components';
+
+<Box>
+  <Checkbox id="checkbox" name="checkbox" />
+  <FormLabel htmlFor="checkbox">Some label here</FormLabel>
+</Box>;
+```
+
+becomes:
+
+```jsx static=true
+import { Checkbox } from '@contentful/f36-components';
+
+<Checkbox id="checkbox" name="checkbox">
+  Some label here
+</Checkbox>;
+```
+
+If you only use `CheckboxField` component, please check the [Field components](#field-components) section.
+
+### RadioButton
+
+In the version 4 we renamed the `RadioButton` to `Radio`, and we removed `RadioButtonField` component. Now instead of doing:
+
+```jsx static=true
+import { RadioButton } from '@contentful/forma-36-react-components';
+
+<RadioButton name="someOption" id="radio" labelText="some label" />;
+```
+
+_Note: This example would render only the input and the `labelText` would be set as the `aria-label`._
+
+You will do the following:
+
+```jsx static=true
+import { Radio } from '@contentful/f36-components';
+
+// This would render the exact same as on version 3
+<Radio name="someOption" id="radio" aria-label="some label" />
+
+// You can also pass the label and help text to the radio
+<Radio name="someOption" id="radio" helpText="Some help text">
+  This label will be rendered at the side of the input
+</Radio>
+```
+
+We also introduced API changes for the radio. This is an overview of the renamed properties:
+
+- `required` was renamed to `isRequired`;
+- `labelText` should be replaced by `aria-label` when not passing a label to the input;
+- `checked` was renamed to `isChecked`;
+- `disabled` was renamed to `isDisabled`.
+
+#### How to migrate your RadioButton components
+
+Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `RadioButton` component to the new version:
```

Comment:
To migrate your `RadioButton` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):